java/src/main/java/com/logicalclocks/hsfs/engine/SparkEngine.java

```diff
@@ -255,10 +255,10 @@ public void registerHudiTemporaryTable(HudiFeatureGroupAlias hudiFeatureGroupAli
    * @param query Query Object
    * @param queryReadOptions Additional read options as key-value pairs, defaults to empty Map
    * @param writeOptions Additional write options as key-value pairs, defaults to empty Map
-   * @param saveMode org.apache.spark.sql.saveMode: Append, Overwrite, ErrorIfExists, Ignore
+   * @param saveMode com.logicalclocks.hsfs.saveMode: Append, Overwrite, ErrorIfExists, Ignore
```

Comment:
This is wrong. `SaveMode` is from the `org.apache.spark.sql` package. What was there was correct.